.github/workflows/tests.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [ '10.x','12.x', '14.x', '16.x' ]
+        node: [ '12.x', '14.x', '16.x', '17.x', '18.x' ]
```

Comment:
I'm not sure we need 17 _and_ 18 here. Do you have a known use-case running on 17? Else lets just stick to the LTS releases

Reply:
Also, 12 is end of life, so lets drop 12?

Reply:
Both sound good to me (17 goes EOL at the end of the month anyways).